packages/@webex/plugin-cc/src/cc.ts

```diff
@@ -118,6 +123,9 @@ export default class ContactCenter extends WebexPlugin implements IContactCenter
           const agentConfig = new AgentConfig(agentId, this.$webex, this.httpRequest);
           this.agentConfig = await agentConfig.getAgentProfile();
           this.$webex.logger.log(`file: ${CC_FILE}: agent config is fetched successfully`);
+          // TODO: We need to emit an event when it is successful
+          /* eslint-disable consistent-return */
+          await this.defaultReLogin();
```

Comment:
**Note**

This needs to be handled, waiting for Ravi's PR as I think that is required for us to handle this event.
Although, I'm not a 100% sure. But, open to any suggestions on this - might need a discussion.